### ISSUE TITLE
InvocationTargetException

### ISSUE THREAD

**Issue by javalama** [OpenHFT/Chronicle-Map#13]
Test case:

```
            new File("logs").mkdir();
            File file = new File("logs/log-test");

            persistentMap = ChronicleMapBuilder.of(byte[].class, byte[].class)
                    .createPersistedTo(file);
```

The second time I run the above code, I get an InvocationTargetException, which is caused by:

java.lang.NoClassDefFoundError: org/xmlpull/v1/XmlPullParserFactory

I'm on Java 8u25, latest OSX.

**Comment by BoundedBuffer** [comment]
can you confirm you are using version 2.0.22rc2 <http://search.maven.org/#artifactdetails%7Cnet.openhft%7Cchronicle-map%7C2.0.22rc2%7Cjar> 

I’ve just taken your example and created the following test case 

net.openhft.chronicle.map.ChronicleMapTest#testByteArrayPersistenceFileReuse

The test passes.

Rob

**Comment by javalama** [comment]
Yes, 2.0.22rc2. I also had to add lang-6.4.21.jar.

Did you test this on OS X ?

**Comment by javalama** [comment]
Note that my application already use XStream and XPP3. Could that cause problems (given the error noting org/xmlpull/v1/XmlPullParserFactory) ?

**Comment by BoundedBuffer** [comment]
I am using a mac OS X  and also our team city agents passed this test case on CentOS and Windows.

**Comment by BoundedBuffer** [comment]
>Note that my application already use XStream and XPP3. Could that cause problems (given the error noting org/xmlpull/v1/XmlPullParserFactory) ?


possibly  - can you try running with 

<dependency>
    <groupId>com.thoughtworks.xstream</groupId>
    <artifactId>xstream</artifactId>
    <version>1.4.7</version>
</dependency>


<dependency>
    <groupId>org.codehaus.jettison</groupId>
    <artifactId>jettison</artifactId>
    <version>1.3.6</version>
</dependency>

<dependency>
    <groupId>org.ops4j.pax.url</groupId>
    <artifactId>pax-url-aether</artifactId>
    <version>2.2.0</version>
</dependency>

**Comment by javalama** [comment]
Problem solved, I already had xpp3_min-1.1.4c.jar on the class path. However, Chronicle-Map requires xpp3-1.1.4c.jar.

Sorry about the noise.
